src/Microsoft.Health.Dicom.Core/Features/Diagnostic/LogForwarder.cs

```diff
@@ -0,0 +1,47 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using EnsureThat;
+using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.Health.Dicom.Core.Features.Model;
+
+namespace Microsoft.Health.Dicom.Core.Features.Diagnostic;
+
+/// <summary>
+/// A log forwarder which sets a property designating a log to be forwarded.
+/// </summary>
+public static class LogForwarder
```

Comment:
This seems like a extension method to log trace. Should we call LogForwarderExtensions